src/openklant/conf/api.py

```diff
@@ -4,7 +4,7 @@
 # Remove the reference - we don't have a single API version.
 del API_VERSION  # noqa
 
-KLANTINTERACTIES_API_VERSION = "0.3.0"
+KLANTINTERACTIES_API_VERSION = "0.3.1"
```

Comment:
we added a new query param, so let's make it a minor version bump
```suggestion
KLANTINTERACTIES_API_VERSION = "0.4.0"
```